xonsh/completers/bash.py

```diff
@@ -1,125 +1,10 @@
-import os
-import shlex
-import pathlib
 import builtins
-import subprocess
 
-import xonsh.platform as xp
-
-from xonsh.completers.path import _quote_paths
-
-BASH_COMPLETE_SCRIPT = r"""
-{source}
-
-# Override some functions in bash-completion, do not quote for readline
-quote_readline()
-{{
-    echo "$1"
-}}
-
-_quote_readline_by_ref()
-{{
-    if [[ $1 == \'* ]]; then
-        # Leave out first character
-        printf -v $2 %s "${{1:1}}"
-    else
-        printf -v $2 %s "$1"
-    fi
-
-    [[ ${{!2}} == \$* ]] && eval $2=${{!2}}
-}}
-
-
-function _get_complete_statement {{
-    complete -p {cmd} 2> /dev/null || echo "-F _minimal"
-}}
-
-_complete_stmt=$(_get_complete_statement)
-if echo "$_complete_stmt" | grep --quiet -e "_minimal"
-then
-    declare -f _completion_loader > /dev/null && _completion_loader {cmd}
-    _complete_stmt=$(_get_complete_statement)
-fi
-
-_func=$(echo "$_complete_stmt" | grep -o -e '-F \w\+' | cut -d ' ' -f 2)
-declare -f "$_func" > /dev/null || exit 1
-
-echo "$_complete_stmt"
-COMP_WORDS=({line})
-COMP_LINE={comp_line}
-COMP_POINT=${{#COMP_LINE}}
-COMP_COUNT={end}
-COMP_CWORD={n}
-$_func {cmd} {prefix} {prev}
-
-for ((i=0;i<${{#COMPREPLY[*]}};i++)) do echo ${{COMPREPLY[i]}}; done
-"""
+from xonsh.completers import bash_completion
```

Comment:
This import should be `from completers.bash_completions import complete_from_bash`